www/src/Pages/PinMapping.jsx

```diff
@@ -236,7 +271,12 @@ export default function PinMappingPage() {
 												max={boards[selectedBoard].maxPin}
 												isInvalid={buttonMappings[button].error}
 												onChange={(e) => handlePinChange(e, button)}
+												onFocus={() => setSelectedButton(button)}
```

Comment:
Unsure if this is used

Reply:
Oh this is not!
It was for something else that I was planning and it didn't work out. This will be cleaned.

Reply:
Cleaned up